### DISCUSSION TITLE
Add time threshold for status alerts

### DISCUSSION THREAD

**Discussion by MorganMLGman** [henrygd/beszel#264] [Ideas]
Hey, 

First of all I would like to thank you for this project, at one time I was working on something similar but had too little time to actively develop it. 

I would like to suggest adding a time threshold for status alerts, as is the case with other alerts. 
I live in the countryside so my internet provider occasionally has problems of a few seconds or so, which 99% of the time go away in less than a minute. Therefore, adding a threshold to the status alerts would be great to get rid of cases where the machine goes down and up in a matter of seconds.

**Comment by henrygd** [comment]
This is on my list to add at some point.

Closing this discussion as there's an existing one here: #215
